internal/controller/services/monitoring/monitoring_controller_support.go

```diff
@@ -216,6 +218,34 @@ func addTracesTemplateData(templateData map[string]any, traces *serviceApi.Trace
 	return nil
 }
 
+// Images can be overridden via environment variables, with defaults based on platform.
+func addImageURLs(rr *odhtypes.ReconciliationRequest, templateData map[string]any) {
+	templateData["KubeRBACProxyImage"] = getImageURL(
+		"RELATED_IMAGE_OSE_KUBE_RBAC_PROXY_IMAGE",
+		"quay.io/brancz/kube-rbac-proxy:v0.20.0",
+		"registry.redhat.io/openshift4/ose-kube-rbac-proxy-rhel9:v4.17",
+		rr.Release.Name,
+	)
+	templateData["PromLabelProxyImage"] = getImageURL(
+		"RELATED_IMAGE_OSE_PROM_LABEL_PROXY_IMAGE",
```

Comment:
Did we inform DevOps about this new image?

RELATED_IMAGE_OSE_KUBE_RBAC_PROXY_IMAGE is already set in the product, but the second one is not.